docs/src/main/paradox/release-notes/10.2.x.md

```diff
@@ -1,27 +1,116 @@
+ * Server-side HTTP pipelining was disabled by default. Common HTTP clients do not support it by default or have it disabled, so to reduce complexity in the common code paths
+   in was disabled by changing the `akka.http.server.pipelining-limit` to `1`.
+
+### Better support for upcoming Scala versions
+
+Scala 2.13 deprecated "adapted arguments" which were heavily used in the routing DSL with the magnet pattern. It allowed that you could have a method with a single argument to
+take any number of arguments in a typesafe fashion by relying on the Scala compiler to convert those arguments to a tuple of arguments. Since a while we generally try to reduce our usage
+of implicits and the magnet pattern to avoid complex compile error messages, long compile times, and complexity. With Akka Http 10.2 we made another effort to reduce our dependency on that
+language feature by providing a set of code-generated overloads for the methods in question to pave our way for upcoming Scala versions.
+
+### Client: configurable setting overrides per target host
+
+It is now possible to set overrides for client settings per target host @ref[via configuration](../client-side/configuration.md#per-host-overrides). For example, if you use Akka HTTP client
+to access different services, it is now possible to tweak pool and connection parameters like the maximum number of concurrent connections (`max-connections`) on a per-target-host-basis.
+Thanks to [David Knapp / @Falmarri](https://github.com/Falmarri) for providing an initial implementation of that feature a long time ago.
+
+### Client: custom client DNS resolution transport
+
+By default, host name resolution is done in the IO layer when a connection attempt is made. In some cases, it can make sense to be able to customize the resolution, for example if
+there are multiple ip addresses returned for a DNS query, this could be used to implement custom DNS load balancing schemes. For that purpose, a new `ClientTransport.withCustomResolver` was
+introduced that allows providing custom resolution for each connection attempt.
+
+See the section about @ref[custom client DNS resolution schemes](../client-side/client-transport.md#custom-host-name-resolution-transport) for more information.
+
+### Performance improvements
+
+ * Better performance for big client pools: Big client pools with more than 500 connections (`max-connections = 500` or more) suffered from a bad choice of datastructure with O(n²) access
+   times, so bigger pools spent too much time just sifting through the internal data structures instead of doing useful work. This has been fixed and a nightly benchmark has been set
+   up to track performance going forward.
+ * Cheaper logging in client pools helps pools of all sizes by avoiding logging when DEBUG logging is disabled.
+ * discardEntityBytes for strict entities is now a no-op while before streams were materialized.
+
+### Streamlined HTTPS configuration
+
+TBD
```

Comment:
(perhaps take part of https://github.com/akka/akka-http/blob/master/docs/src/main/paradox/migration-guide/migration-guide-10.2.x.md#configuring-https-connections as starting point?)

Reply:
Yes, good idea.